languages/de-DE/Misc.multids

```diff
@@ -74,6 +79,10 @@ Shortcuts/Input/Up/Hint: Gehe zum vorherigen Element
 Shortcuts/Input/Tab-Left/Hint: Gehe zum vorherigen Tab 
 Shortcuts/Input/Tab-Right/Hint: Gehe zum nächsten Tab
 Shortcuts/SidebarLayout/Hint: Das Layout des rechten Menüs ändern
+Switcher/Subtitle/theme: "Theme" auswählen
+Switcher/Subtitle/layout: "Layout" auswählen
```

Comment:
Drop the quotes here too, Layout is even used without quotes already right above :)